### PR TITLE
Bootstrap4alpha6

### PR DESCRIPTION
**Pull request by agriffard**
See #545 : Update Bootstrap 4 to alpha 6

### CODE REVIEW

**Comment by Jetski5822** [comment]
Whoah, lots of change!

**Comment by Jetski5822** [comment]
@agriffard can you rebase dude?

File: src/Orchard.Cms.Web/Modules/Orchard.Flows/Views/Widget-Flow.Edit.cshtml [hunk 1]
```diff
@@ -6,12 +6,12 @@
     var widgetContentTypes = ContentDefinitionManager.ListTypeDefinitions().Where(t => t.Settings.ToObject<ContentTypeSettings>().Stereotype == "Widget");
 }
 
-<div class="widget widget-editor card @(Model.ContentItem.Number != 0 ? "collapsed" : "")">
+<div class="widget widget-editor card my-1 @(Model.ContentItem.Number != 0 ? "collapsed" : "")">
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Modules/Orchard.Flows/Views/Widget-Flow.Edit.cshtml]: what is `my-1`

File: src/Orchard.Cms.Web/Themes/TheAdmin/Assets/scss/TheAdmin.scss [hunk 2]
```diff
@@ -287,16 +297,4 @@ ul.pager li a {
 
 .col-xl {
     @extend .col-xs-12, .col-sm-12, .col-md-12, .col-lg-12, .col-xs-12;
-}
```
Comment (sebastienros) [inline src/Orchard.Cms.Web/Themes/TheAdmin/Assets/scss/TheAdmin.scss]: Are you sure? Did you see the actual issue before doing this?

**Comment by sebastienros** [comment]
You did a rebase, but the changes from the Flows module appear in your changes, is that normal?

**Comment by sebastienros** [comment]
Got it, you actually didn't rebase, but merged master. That will work, even though it's ugly right now.

File: src/Orchard.Cms.Web/Modules/Orchard.Flows/Views/Widget-Flow.Edit.cshtml (hunk 1 above)
Comment (agriffard) [inline src/Orchard.Cms.Web/Modules/Orchard.Flows/Views/Widget-Flow.Edit.cshtml]: https://v4-alpha.getbootstrap.com/utilities/spacing/ : margin top + bottom

**Pull request merged**